modules/ocf_stats/manifests/prometheus.pp

```diff
@@ -1,5 +1,9 @@
 # prometheus daemon config
 class ocf_stats::prometheus {
+  # The list of nodes to monitor-- for now, monitor all hosts.
+  $nodes_query = '["from", "nodes", ["=", "expired", null]]'
+  $nodes = sort(puppetdb_query($nodes_query).map |$value| { $value["certname"] })
+
   class { '::prometheus':
     version        => '2.0.0',
     extra_options  => '--web.listen-address="127.0.0.1:9090" --web.external-url=http://127.0.0.1/prom',
```

Comment:
I'd also call this `/prometheus` instead of just `/prom` since it's marginally more descriptive for someone reading this (and the future nginx config, I imagine) in the future.